SIDFactoryII/source/runtime/editor/components/component_table_row_elements.cpp

```diff
@@ -346,6 +353,11 @@ namespace Editor
 				{
 					int base_data_offset = row * m_DataSource->GetColumnCount();
 
+					if (m_HighlightRowOfSelectedSongIndex)
```

Comment:
Dead code. Need?

Reply:
I'll update this so that it reflects that I need to implement the highlighting of the selected song feature! (Which is for the init table, so that it is evident which lines is relative to the current selected song)